scripts/pylib/twister/twisterlib/handlers.py

```diff
@@ -40,6 +40,7 @@
 logger.setLevel(logging.DEBUG)
 
 SUPPORTED_SIMS = ["mdb-nsim", "nsim", "renode", "qemu", "tsim", "armfvp", "xt-sim", "native"]
+SUPPORTED_SIMS_IN_PYTEST = ['native', 'qemu']
```

Comment:
is this change related to the PR?

Reply:
I moved it to avoid circular importing. It is in this PR, because when added annotations (I know, that annotations could be in separate PR), circular importing happens